e2e/tests/notebook/notebook-download.spec.ts

```diff
@@ -0,0 +1,88 @@
+import NotebookDownloadModal from 'app/page/notebook-download-modal';
+import WorkspaceDataPage from 'app/page/workspace-data-page';
+import {makeRandomName} from 'utils/str-utils';
+import {findWorkspace, signIn} from 'utils/test-utils';
+import {waitForFn} from 'utils/waits-utils';
+
+describe('Jupyter notebook download test', () => {
+
+  beforeEach(async () => {
+    await signIn(page);
+  });
+
+  const testDownloadModal = async (modal: NotebookDownloadModal): Promise<void> => {
+    let downloadBtn = await modal.getDownloadButton();
+    expect(await downloadBtn.getProperty('disabled')).toBeTruthy();
```

Comment:
This check always pass even for `disabled: false` because `getProperty()` returns a `JSHandle` object. 

This code checks properly:
```
   let downloadBtn = await modal.getDownloadButton();
   // sleep 2 seconds because state of the button changes. polling for "disable: true" would be a nicer solution. Without sleep, `disabled` is `false` after modal is opened because the check happened too soon.
    await page.waitFor(2000);
    expect(await getPropValue<boolean>(downloadBtn, 'disabled')).toBe(true);

    await modal.clickPolicyCheckbox();

    expect(await getPropValue<boolean>(downloadBtn, 'disabled')).toBe(false);
```

Reply:
Oops thanks, fixed.